README.md

```diff
@@ -68,23 +68,28 @@ If you don't currently have Rails installed (or are on Windows), Cloud9 makes th
 
 ### Docker
 
+<<<<<<< HEAD
```

Comment:
This was popping up in the other PR too, can you make the necessary changes here?